vllm/model_executor/models/phi3v.py

```diff
@@ -389,15 +373,31 @@ def _apply_hf_processor(
         processed_outputs['input_ids'] = token_ids
         return processed_outputs
 
-    def _get_dummy_mm_kwargs(
+    def _get_dummy_mm_inputs(
         self,
         mm_counts: Mapping[str, int],
-    ) -> MultiModalKwargs:
-        return dummy_mm_kwargs_for_phi3v(self.ctx, mm_counts)
+    ) -> ProcessorInputs:
+        num_images = mm_counts["image"]
+
+        data = dummy_image_for_clip(
+            CLIP_VIT_LARGE_PATCH14_336_CONFIG,
+            num_images,
+            image_width_override=MAX_IMAGE_FEATURE_SIZE_WIDTH,
+            image_height_override=MAX_IMAGE_FEATURE_SIZE_HEIGHT,
+        )
+
+        hf_processor = self._get_hf_processor()
+        image_tokens = hf_processor.img_tokens  # type: ignore
+
+        return ProcessorInputs(
+            prompt_text=image_tokens[:num_images],
```

Comment:
```suggestion
            prompt_text="".join(image_tokens[:num_images]),
```
Seems that this broke phi3v multi-images inference.

Reply:
I applied this fix but got a placeholder mismatch error for Phi-3-Vision.

Reply:
Seems that this is due to the incorrect replacement, it can work after this modification (removed `get_max_phi3v_image_tokens`):
```python
def create_metadata_for_phi3v(
        ctx: InputProcessingContext) -> MultiModalProcessingMetadata:
    return {
        "image":
        ModalityProcessingMetadata(prompt_repls=[
            PromptReplacement(
                target=[_IMAGE_TOKEN_ID],
                replacement=[_IMAGE_TOKEN_ID],
            ),
        ]),
    }
```

Reply:
This just disables prompt replacement, feels like a hack 😅

Reply:
Actually, this is still not really correct as it fails to detect distinguish between image tokens from different images. We need to get the exact tokens counts for each image.

Reply:
Hmmm, let me think a simpler way to compute the image tokens for metadata initialization...

Reply:
I have pushed a commit that adds back the old methods for computing the image size, see if it works!